packages/aws-serverless/src/awslambda-auto.ts

```diff
@@ -21,7 +23,9 @@ if (lambdaTaskRoot) {
     ),
   });
 
-  Sentry.tryPatchHandler(lambdaTaskRoot, handlerString);
+  if (typeof require !== 'undefined') {
```

Comment:
Fwiw, this is the same check we use in our `isCjs` utility within the SDK. I just can't import it here because this doesn't play well with the lambda layer being bundled into the index file imported on top.